src/openstudio_lib/GeometryEditorView.cpp

```diff
@@ -487,6 +519,14 @@ void FloorspaceEditor::translateExport() {
   if (model) {
     m_exportModel = *model;
     m_exportModelHandleMapping = rt.handleMapping();
+
+    // manually add mappings between current model and new model for Site, Facility, and Building objects
+    m_exportModelHandleMapping[m_model.getUniqueModelObject<model::Site>().handle()] = m_exportModel.getUniqueModelObject<model::Site>().handle();
+    m_exportModelHandleMapping[m_model.getUniqueModelObject<model::Facility>().handle()] =
+      m_exportModel.getUniqueModelObject<model::Facility>().handle();
+    m_exportModelHandleMapping[m_model.getUniqueModelObject<model::Building>().handle()] =
+      m_exportModel.getUniqueModelObject<model::Building>().handle();
```

Comment:
Why is this needed? I'm not doubting it isn't, just curious.

Reply:
This one is pretty weird.  Basically we don't have site, facility, or building objects in the FloorspaceJS file so we don't have anywhere to store the corresponding OpenStudio Model handles.

Reply:
I did leave some comments in the OS SDK: https://github.com/macumber/OpenStudio/blob/ac01edda9d24618b9c552f623e0a5c92bdd5de86/src/model/ThreeJSReverseTranslator.hpp#L67